src/Discovery/DiscoveryTagViewer.tsx

```diff
@@ -139,12 +141,42 @@ const DiscoveryTagViewer: React.FunctionComponent<DiscoveryTagViewerProps> = (pr
                 xl={6}
                 xxl={4}
               >
-                <h5 className='discovery-header__tag-group-header'>{categoryDisplayName}</h5>
-                { tags }
-              </Col>
-            );
-          })
-        }
+                <table>
```

Comment:
all my comments above regarding to src/Discovery/DiscoveryDropdownTagViewer.tsx also applies to here